### ISSUE TITLE
Setting to turn window preview into a list, if there are too many windows open.

### ISSUE THREAD

**Issue by rzvc** [home-sweet-gnome/dash-to-panel#1649]
Hi,

First of all, thank you for this extension. It's absolutely great.

The issue:
Whenever I have too many windows open of the same type, the window preview is so long I have to scroll through it. This can become a little confusing for some type of windows (like all-text windows).

In those situations it would be nice for the preview to turn off and instead show a list of windows (just the titles), stacked one on top of the other, like it happens on Windows.

A setting to switch between the scrolling behaviour and the list behaviour, when the previews extend past the edge of the screen, would be great.

Cheers.

**Comment by hbh7** [comment]
Agreed, this would be a great feature!

**Comment by hbh7** [comment]
Nope, this would definitely still be nice. Not really "stale", just not worked on by anyone yet as far as I know.

**Comment by hbh7** [comment]
As far as I know, this still isn't a thing yet :(
